.github/workflows/GoCliTests.yaml

```diff
@@ -0,0 +1,27 @@
+name: Run Go CLI unit tests and Godog functional tests
+
+on:
+  push:
```

Comment:
Isn't there a better way to do this to monitor only changes to the goinstant directory? Or is this why we need a git submodule for the GO CLI?

Reply:
As per discussions, this can be done in a future task.